P/PGPLOT/build_tarballs.jl

```diff
@@ -5,30 +5,54 @@ using BinaryBuilder, Pkg
 name = "PGPLOT"
 version = v"5.2"
 
-sources = [ArchiveSource("ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz", "a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4")]
+sources = [
+    ArchiveSource("ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz", "a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4"),
+    DirectorySource("./bundled"),
+]
 
 script = raw"""
 cd $WORKSPACE/srcdir
+
+if [[ "${target}" == *-apple-* ]]; then
+    cd pgplot
+    for f in ../patch-*;
+    do
+    patch -p0 <$f;
+    done
+    mkdir sys_darwin
+    cp ../bb.conf ./sys_darwin/bb.conf
+    cd ..
+fi
```

Comment:
I thought I left a comment about this, but looks like I didn't save
```suggestion
if [[ "${target}" == *-apple-* ]]; then
    pushd pgplot
    for f in ../patch-*; do
        atomic_patch -p0 "${f}"
    done
    mkdir sys_darwin
    cp ../bb.conf ./sys_darwin/bb.conf
    popd
fi
```